tests/conftest.py

```diff
@@ -344,7 +344,11 @@ def pytest_collection_modifyitems(session, config, items):
                     log.debug(f"Test: {item} will be skipped due to {skip_condition}")
                     items.remove(item)
                     continue
-            if skipif_upgraded_from_marker:
+            if (
+                skipif_upgraded_from_marker
+                and config.ENV_DATA.get("platform", "").lower()
```

Comment:
```suggestion
                and ocsci_config.ENV_DATA.get("platform", "").lower()
```
This should fix the pr_check error.